lib/absinthe/blueprint/document/operation.ex

```diff
@@ -18,6 +18,7 @@ defmodule Absinthe.Blueprint.Document.Operation do
     # Populated by phases
     flags: %{},
     schema_node: nil,
+    complexity: nil,
```

Comment:
I didn't like to set the complexity in case someone wanted to make use of the complexity in another phase.